andela/activo-web
# #157469427 Design asset category cards

#### What does this PR do?
Setup the component for the display of asset category cards

### Description of task to be completed?
- create dashboard component
- create globals, mixins and variables scss files
- import DinPro font file
- write test for Assest category cards

#### How should this be manually tested?
- After cloning the repo, run `yarn install` to install the necessary packages
- Run `yarn start:dev` to start the webpack dev-server on `port 3000`.
- Open web browser to `localhost:3000` to see the dashboard component

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#157469427](https://www.pivotaltracker.com/story/show/157469427)

#### Screenshots
<img width="1280" alt="screen shot 2018-05-28 at 3 47 48 pm" src="https://user-images.githubusercontent.com/24798364/40619844-8612e290-628e-11e8-870b-7e5ffb718afd.png">
